content/models/index.mdx

```diff
@@ -188,7 +195,15 @@ For example, `opensource(reasoning,throughput,Brand)` would return structured da
 You can also specify a schema for the output (styled like TypeScript):
 
 ```
-r1(output:{ address: string, phoneNumber: number, name: string },cost)
+r1(output:{ address: string, phoneNumber: number, name: string }, cost)
```

Comment:
This should follow the same convention as functions.do simplified schema, but probably with a more concise syntax as practically speaking, 99% of the time everything should either be a string or an enum